SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKDefaultLaunchConfigGenerator.cpp

```diff
@@ -0,0 +1,172 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+
+#include "SpatialGDKDefaultLaunchConfigGenerator.h"
+
+#include "SpatialGDKEditorSettings.h"
+
+#include "Serialization/JsonWriter.h"
+#include "Misc/FileHelper.h"
+
+DEFINE_LOG_CATEGORY(LogSpatialGDKDefaultLaunchConfigGenerator);
+
+#define LOCTEXT_NAMESPACE "SpatialGDKDefaultLaunchConfigGenerator"
+
+using namespace SpatialGDK;
+
+namespace
+{
+bool WriteFlagSection(TSharedRef<TJsonWriter<>> Writer, const FString& Key, const FString& Value)
+{
+	Writer->WriteObjectStart();
```

Comment:
This class appears to be using tabs whereas I think we normally use spaces?

Reply:
No